rules/REQUEST-934-APPLICATION-ATTACK-GENERIC.conf

```diff
@@ -28,7 +28,7 @@ SecRule TX:DETECTION_PARANOIA_LEVEL "@lt 1" "id:934012,phase:2,pass,nolog,tag:'O
 # - funcster: __js_function
 #
 # See:
-# https://opsecx.com/index.php/2017/02/08/exploiting-node-js-deserialization-bug-for-remote-code-execution/
+# https://opsecx.com/index.php/2017/02/08/exploiting-platform-node-js-deserialization-bug-for-remote-code-execution/
```

Comment:
This change looks that was part of an automated change. Is the url working?

Reply:
Apoligies, fixed that

Reply:
Maybe it's me, but I don't see the fix... 🤷

Reply:
Getting a _404 Not Found_ here, for that link..

Reply:
yep really sorry about that, ive finally reverted it all and made sure i havent change the urls in the last commit